packages/dd-trace/src/appsec/iast/taint-tracking/index.js

```diff
@@ -0,0 +1,98 @@
+'use strict'
+
+const { storage } = require('../../../../../datadog-core')
+const iastContextFunctions = require('../iast-context')
+const log = require('../../../log')
+const { enableRewriter, disableRewriter } = require('./rewriter')
+const IAST_TRANSACTION_ID = Symbol('_dd.iast.transactionId')
+
+let TaintedUtils
+try {
```

Comment:
This code with try and catch, wouldn't it be better in `enableTaintTracking` function? 
This is a personal opinion, but I don't like to see logic out of the methods if it is not 100% necessary.